com/positionsaufabschlag.go

```diff
@@ -0,0 +1,18 @@
+package com
+
+import (
+	"github.com/hochfrequenz/go-bo4e/enum/aufabschlagstyp"
+	"github.com/hochfrequenz/go-bo4e/enum/waehrungseinheit"
+	"github.com/shopspring/decimal"
+)
+
+// PositionsAufAbschlag Differenzierung der zu betrachtenden Produkte anhand der preiserhöhenden (Aufschlag) bzw. preisvermindernden
+// (Abschlag) Zusatzvereinbarungen, die individuell zu einem neuen oder bestehenden Liefervertrag abgeschlossen werden
+// können. Es können mehrere Auf-/Abschläge gleichzeitig ausgewählt werden.
+type PositionsAufAbschlag struct {
+	Bezeichnung          string                            `json:"bezeichnung,omitempty" validate:"required"`          // Bezeichnung des Auf-/Abschlags
+	Beschreibung         string                            `json:"beschreibung,omitempty" validate:"required"`         // Beschreibung zum Auf-/Abschlags
+	AufAbschlagstyp      aufabschlagstyp.AufAbschlagstyp   `json:"aufabschlagstyp,omitempty" validate:"required"`      // Typ des AufAbschlages
```

Comment:
```suggestion
	AufAbschlagstyp      aufAbschlagstyp.AufAbschlagstyp   `json:"aufabschlagstyp,omitempty" validate:"required"`      // Typ des AufAbschlages
```
siehe https://github.com/Hochfrequenz/BO4E-dotnet/blob/e148b1e425322b5a09b5d2cf1716710b5119ce89/BO4E/COM/PositionsAufAbschlag.cs#L30